_posts/2023-02-20-example-mapping-in-practice.markdown

```diff
@@ -0,0 +1,179 @@
+6. The board is continually refined as questions are answered, new rules written and examples provided
+7. To ensure that we have something solid to start on, I try to keep focus on one Rule at a time to ensure that sufficient examples are required and continually ask "Dev team, are you happy you can start building this" and "Product Owner, are you happy with this outcome"
+
+At the end of the session the goal is to at least have a starting point for the User Story development and a plan to follow up any additional sessions requried.
+
+<br />
+
+# The Opposition
+
+There's obviouslt many ways to specify and communicate requirements, so lets stack Example Mapping up against the opposition.
+
+## The Func Spec
+
+![Example Mapping](/img/blog/examplemapping/funcspec.jpg)
+
+Ahhh the old faithful! What can go wrong? Well...let me tell you...
+As 'Engineer Liason' [Tom Smykowski](https://www.imdb.com/title/tt0151804/characters/nm0726223) told us in the Movie Office Space _"I deal with the god damn customers so the engineers don't have to. I have people skills; I am good at dealing with people. Can't you understand that? What the hell is wrong with you people?"_
+
+So according to Tom, having the engineers talk directly with the users is just not right, so instead a specialist role exists. Sounds simple right? But here is what really happens:
+
+- The BA talks to 'The Business', converts their business speak into tech requirements and places them in a nice document
+- The developer picks up the func spec and codes their interpretation
+- The tester then picks up the spec and tests the code to the letter of the document
+- The developer and tester get into a massive argument
+- The BA gets involved and tells them that they are both wrong
+- 'The Business' then come along and tell them all that the spec does not reflect what they wanted
+
+And all that is based on the fact that the developer even reads the spec.
+
+Essentially there is a large up front investment which does not involve the right people, everyone then communicates through a document....working away in silos.
+
+## The "She'll be right"
+
+You've all seen this one...the Jira Card with a title and no body!
+![Example Mapping](/img/blog/examplemapping/emptyjiracard.png)
+
+The Scrum master/BA/product owner/developer write's down a user story and adds it to the backlog. Nobody writes any acceptance criteria because "It's easy right".
+
+In this scenario, the following usually happens:
+
+- Developer writes the feature they want
+- Tester tests the feature they expect
+- Argument happens
+- Someone FINALLY brings in the product owner and the explain what they actually want
+
+There's always more to the story than meets the eye. Subsequenely the code gets thrown in the bin and story development restarts.
+
+If only there was some way of that short conversation happening between developer, tester and product owner beforehand :wink:
```

Comment:
I really like this - you're getting the reader to think and make them join the dots with Example Mapping.

I like your summary in the previous section:
> Essentially there is a large up front investment which does not involve the right people, everyone then communicates through a document....working away in silos.
But can you edit it to use the same approach to get the reader to think?

Repetition...repetition...repetition

Reply:
Added to the previous bit `"Essentially there is a large up front investment which does not involve the right people, everyone then communicates through a document....working away in silos. Perhaps working through 'The Spec' in collaboration might work better?"`